feat(containsURL): Add Contains URL validator (#1886)
validatorjs/validator.js: README.md

> **containsUrl(str [, options])**            | checks if the string contains an email using default isURL() options. (Use {deep: true} as options to check for uncommon url formats like http://[::192.9.5.5]/ipng)

Reviewer: ```suggestion
**containsUrl(str [, options])**            | checks if the string contains a URL using default isURL() options. (Use {deep: true} as options to check for uncommon URL formats like http://[::192.9.5.5]/ipng)
```
